esrally/track/track.py

```diff
@@ -701,7 +701,7 @@ class OperationType(Enum):
     CompositeAgg = (18, AdminStatus.No, serverless.Status.Public)
     WaitForCurrentSnapshotsCreate = (19, AdminStatus.No, serverless.Status.Internal)
     Downsample = (20, AdminStatus.No, serverless.Status.Internal)
-    Esql = (21, AdminStatus.No, serverless.Status.Blocked)
+    Esql = (21, AdminStatus.No, serverless.Status.Internal)
```

Comment:
ESQL is a public serverless API. Let's go ahead and mark it public here as well.